src/test/java/seedu/address/logic/parser/CreateCommandParserTest.java

```diff
@@ -66,32 +66,32 @@ public void parse_allFieldsPresent_success() {
         // whitespace only preamble
         assertParseSuccess(parser, PREAMBLE_WHITESPACE + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
                 + ADDRESS_DESC_BOB + NETWORTH_DESC_BOB + MEETING_TIME_DESC_BOB + DESCRIPTION_DESC_BOB
-                + TAG_DESC_BOB, new CreateCommand(expectedPerson));
+                + TAG_DESC_AMY, new CreateCommand(expectedPerson));
 
         // multiple names - last name accepted
         assertParseSuccess(parser, NAME_DESC_AMY + NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB
                 + ADDRESS_DESC_BOB + NETWORTH_DESC_BOB + MEETING_TIME_DESC_BOB + DESCRIPTION_DESC_BOB
-                + TAG_DESC_BOB, new CreateCommand(expectedPerson));
+                + TAG_DESC_AMY, new CreateCommand(expectedPerson));
 
         // multiple phones - last phone accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_AMY + PHONE_DESC_BOB + EMAIL_DESC_BOB
                 + ADDRESS_DESC_BOB + NETWORTH_DESC_BOB + MEETING_TIME_DESC_BOB + DESCRIPTION_DESC_BOB
-                + TAG_DESC_BOB, new CreateCommand(expectedPerson));
+                + TAG_DESC_AMY, new CreateCommand(expectedPerson));
 
         // multiple emails - last email accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_AMY + EMAIL_DESC_BOB
                 + ADDRESS_DESC_BOB + NETWORTH_DESC_BOB + MEETING_TIME_DESC_BOB + DESCRIPTION_DESC_BOB
-                + TAG_DESC_BOB, new CreateCommand(expectedPerson));
+                + TAG_DESC_AMY, new CreateCommand(expectedPerson));
 
         // multiple addresses - last address accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_AMY
                 + ADDRESS_DESC_BOB + NETWORTH_DESC_BOB + MEETING_TIME_DESC_BOB + DESCRIPTION_DESC_BOB
-                + TAG_DESC_BOB, new CreateCommand(expectedPerson));
+                + TAG_DESC_AMY, new CreateCommand(expectedPerson));
 
         // multiple networths - last networth accepted
         assertParseSuccess(parser, NAME_DESC_BOB + PHONE_DESC_BOB + EMAIL_DESC_BOB + ADDRESS_DESC_BOB
                 + NETWORTH_DESC_AMY + NETWORTH_DESC_BOB + MEETING_TIME_DESC_BOB + DESCRIPTION_DESC_BOB
-                + TAG_DESC_BOB, new CreateCommand(expectedPerson));
+                + TAG_DESC_AMY, new CreateCommand(expectedPerson));
```

Comment:
Should be changed back to TAG_DESC_BOB, but doesn't affect the test cases. 
Likewise for the other tests